### PR TITLE
theme: Flavor

### PR DESCRIPTION
**Pull request by xtexChooser**
- [x] I want to publish my theme on Hexo official website.
  - [x] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
  - [x] `name` is unique.
  - [x] `link` URL is correct.
  - [x] `preview` URL is correct.
  - [x] `preview` URL web site is rendered correctly.
  - [x] Add a screenshot to `source/themes/screenshots`.
  - [x] Screenshot filename is same as value of `name`.
  - [x] Screenshot size is `800 * 500`.
  - [x] Screenshot file format is `png`.

### CODE REVIEW

**Comment by jiangtj** [comment]
I found your preview link seems to be broken.

**Comment by xtexChooser** [comment]
> I found your preview link seems to be broken.

what? Its my blog and I can open it correctly

**Review by tomap — APPROVED** [review]

File: source/_data/themes.yml [hunk 1]
```diff
@@ -3579,3 +3579,13 @@
     - Elegant
     - Responsive
     - Clean
+- name: Flavor
+  description: A Simple and Fast Theme for Hexo
+  link: https://github.com/xtexChooser/hexo-theme-flavor
+  preview: https://xtexx.ml/blog/2021-8-23-custom-theme/
```
Comment (stevenjoezhang) [inline source/_data/themes.yml:3585]: 截屏2021-11-03 下午12 45 16

The link returns 404 not found

**Review by stevenjoezhang — CHANGES_REQUESTED** [review]

File: source/_data/themes.yml (hunk 1 above)
Comment (xtexChooser) [reply source/_data/themes.yml]: Thanks, resolved

**Comment by xtexChooser** [comment]
Thanks

**Pull request merged**